torch/_subclasses/fake_tensor.py

```diff
@@ -1030,12 +1243,65 @@ class DispatchCacheInfo:
     Information about the state of the FakeTensor dispatch cache.
     """
 
+    __slots__ = ("hits", "misses", "bypasses", "size")
```

Comment:
Annoying we cant juse use py 3.10 `@dataclass(slots=True)`.. in a few years i guess